packages/react-native-sdk/src/components/participants/ParticipantView.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ComponentType } from 'react';
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native';
+import { StreamVideoParticipant } from '@stream-io/video-client';
+import { theme } from '../../theme';
+import { palette } from '../../theme/constants';
+import {
+  ParticipantNetworkQualityIndicator as DefaultParticipantNetworkQualityIndicator,
+  ParticipantNetworkQualityIndicatorProps,
+} from './ParticipantNetworkQualityIndicator';
+import {
+  ParticipantReaction as DefaultParticipantReaction,
+  ParticipantReactionProps,
+} from './ParticipantReaction';
+import {
+  ParticipantLabel as DefaultParticipantLabel,
+  ParticipantLabelProps,
+} from './ParticipantLabel';
+import { ParticipantVideoFallbackProps } from './ParticipantVideoFallback';
+import {
+  VideoRenderer as DefaultVideoRenderer,
+  VideoRendererProps,
+} from './VideoRenderer';
+
+export type ParticipantVideoType = 'video' | 'screen';
+
+/**
+ * Props to be passed for the Participant component.
+ */
+export type ParticipantViewProps = {
+  /**
+   * The participant that will be displayed
+   */
+  participant: StreamVideoParticipant;
+  /**
+   * The video kind that will be displayed
+   */
+  videoMode: ParticipantVideoType;
+  /**
+   * Any custom style to be merged with the participant view
+   */
+  containerStyle?: StyleProp<ViewStyle>;
```

Comment:
```suggestion
  style?: StyleProp<ViewStyle>;
```

simple change right?!

Reply:
and consistent with other SDKs

Reply:
yup